carma-messenger-core/carma-messenger/CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ project(carma-messenger)
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
 find_package(catkin REQUIRED COMPONENTS
- message
```

Comment:
lets add cpp_message here and in the package.xml to keep the dependency chain consistent with what we had before.